<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="8714" started-at="2015-09-01T06:42:38Z" finished-at="2015-09-01T06:42:47Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="8714" started-at="2015-09-01T06:42:38Z" finished-at="2015-09-01T06:42:47Z">
      <class name="test.PO">
        <test-method status="PASS" signature="CreateWebDriver()[pri:0, instance:test.PO@224aed64]" name="CreateWebDriver" is-config="true" duration-ms="5576" started-at="2015-09-01T09:42:38Z" finished-at="2015-09-01T09:42:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CreateWebDriver -->
        <test-method status="FAIL" signature="Test2()[pri:0, instance:test.PO@224aed64]" name="Test2" duration-ms="3091" started-at="2015-09-01T09:42:44Z" finished-at="2015-09-01T09:42:47Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at test.PDCMainPage.CheckTitle(PDCMainPage.java:29)
	at test.PO.Test2(PO.java:37)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:648)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:834)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1142)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1176)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1101)
	at org.testng.TestNG.run(TestNG.java:1009)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test2 -->
        <test-method status="PASS" signature="CloseWebDriver()[pri:0, instance:test.PO@224aed64]" name="CloseWebDriver" is-config="true" duration-ms="20" started-at="2015-09-01T09:42:47Z" finished-at="2015-09-01T09:42:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CloseWebDriver -->
      </class> <!-- test.PO -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
